_includes/footer.html

```diff
@@ -29,7 +29,7 @@
         Coworking Festival
       </span>
       &bull;
-      <a class="link--footer pl2 pr0" href="/de/impressum">Imprint</a>
+      <a class="link--footer pl2 pr0" href="/de/impressum">Impressum</a>
```

Comment:
its the english footer, so it should stay english. (we don't translate the imprint)